src/Arcus.Testing.Core/Arcus.Testing.Core.csproj

```diff
@@ -28,9 +28,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="[8.0.1,10.0.0)" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[8.0.0,10.0.0)" />
-    <PackageReference Include="Polly" Version="8.3.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="[8.*,10.0.0)" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[8.*,10.0.0)" />
+    <PackageReference Include="Polly" Version="[8.*,9.0.0)">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Made **Polly** a private asset, as this package is only used for internal use/internal choice, and is not something that the consumer should be aware or concern themselves about.